pkg/security/apparmor/validate.go

```diff
@@ -111,9 +111,9 @@ func validateHost(runtime string) error {
 		return errors.New("AppArmor is not enabled on the host")
 	}
 
-	// Check runtime support. Currently only Docker is supported.
-	if runtime != kubetypes.DockerContainerRuntime && runtime != kubetypes.RemoteContainerRuntime {
-		return fmt.Errorf("AppArmor is only enabled for 'docker' and 'remote' runtimes. Found: %q", runtime)
+	// Check runtime support. Currently only CRI is supported.
+	if runtime != kubetypes.RemoteContainerRuntime {
+		return fmt.Errorf("AppArmor is only enabled for 'remote' runtimes. Found: %q", runtime)
```

Comment:
it might sense to remove the whole if conditions, as runtime can only be `remote` in context of kubelet though not sure what values runtime can have in this package

Reply:
+1 to @adisky 's suggestion.

Reply:
+1, removing this check entirely seems reasonable to me.

Reply:
thanks for your suggestion, the whole if conditions has been removed